From 2408ad68c45d5d8432316a37b3aa42707ec9bea2 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Mon, 26 Feb 2018 22:14:28 +0000
Subject: [PATCH] TRITON-180 Move "sdc" zone to new Triton origin image
 Reviewed by: Julien Gilli <julien.gilli@joyent.com> Approved by: Julien Gilli
 <julien.gilli@joyent.com>

---
 targets.json.in | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/targets.json.in b/targets.json.in
index f12e7e5..0a176c8 100644
--- a/targets.json.in
+++ b/targets.json.in
@@ -685,9 +685,10 @@ cat <<EOF
     "build_platform": "20151126T062538Z",
     "image_name": "sdc",
     "image_description": "SDC tools/ops zone",
-    "image_uuid": "fd2cc906-8938-11e3-beab-4359c665ac99",
+    "// image_uuid": "triton-origin-multiarch-15.4.1@1.0.1",
+    "image_uuid": "04a48d7d-6bb5-4e83-8c3b-e60a99e0f48f",
     "pkgsrc": [
-        "mtr-0.82"
+        "mtr-0.86nb3"
     ],
     "repos": [
       {"url": "git@github.com:joyent/sdc-sdc.git"}
-- 
2.21.0

